docs/components/input-controller/_index.md

```diff
@@ -1071,7 +1071,7 @@ if __name__ == '__main__':
     asyncio.run(main())
 ```
 
-### Drive a robot with Four Wheels & a Skid Steer Platform
+### Drive a robot with four wheels & a skid steer platform
```

Comment:
```suggestion
### Drive a robot with four wheels and a skid steer platform
```
[opt] Out of diff but we don't usually use ampersands--if you feel like making this change. It doesn't look like there are any links to this in the docs.